waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/TunnelingMetaStoreClientFactory.java

```diff
@@ -113,7 +113,7 @@ public CloseableThriftHiveMetastoreIface newInstance(HiveConf hiveConf, String n
         .get(localHiveConf);
 
     LOG.info("Metastore URI {} is being proxied to {}", localHiveConf.getVar(HiveConf.ConfVars.METASTOREURIS),
-        localHiveConf.getVar(HiveConf.ConfVars.METASTOREURIS));
+        hiveConf.getVar(HiveConf.ConfVars.METASTOREURIS));
```

Comment:
Oh dear, this looks like it was wrong too, what would this cause to happen?

Reply:
it is just a log message it would say, localhost:1234 is being proxied to localhost:1234, but it is fixed now. ;)